src/components/common/FacebookButton.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import FacebookLogin from 'react-facebook-login';
+import { face_app_key } from '../../constants/constants';
+import { facebookLogin } from '../../actions/sessionActions';
+
+class FacebookButton extends Component {
+
+  render() {
+    return (
+      <FacebookLogin
+        appId={face_app_key}
+        autoLoad={true}
```

Comment:
If it's a boolean, and you want to pass a `true` value, with the prop name is just enough.